controllers/users.js

```diff
@@ -612,7 +614,19 @@ const markUnverified = async (req, res) => {
     });
 
     usersToApplyUnverifiedRole.forEach((id) => {
-      addRolePromises.push(addRoleToUser(id, unverifiedRoleId));
+      addRolePromises.push(
+        addRoleToUser(id, unverifiedRoleId),
+        addGroupRoleToMember({
+          roleid: unverifiedRoleId,
+          userid: id,
+          date: admin.firestore.Timestamp.fromDate(new Date()),
+        }),
+        addLog(
```

Comment:
@shobhan-sundar-goutam Combining multiple database writes into a batch operation can reduce latency and improve performance

Reply:
Batch insertion is done. Can you check it please